packaging/linux/common/khiops

```diff
@@ -3,13 +3,65 @@
 # Run Khiops tool
```

Comment:
Remove "WIP" from the commit message. Also state it clearly that the scope of this change is Linux.

Reply:
IMHO, we also need to change the commit message of commit https://github.com/KhiopsML/khiops/pull/284/commits/43ba4ae6608f556352d2175834e805b255d420a0:

- drop the `WIP` prefix
- add the `on Linux` suffix

Reply:
done